packages/janet.rb

```diff
@@ -0,0 +1,31 @@
+require 'package'
+
+class Janet < Package
+  description 'a functional and imperative programming language.'
+  homepage 'https://janet-lang.org/'
+  version '0.6.0'
+  source_url 'https://github.com/janet-lang/janet/archive/v0.6.0.tar.gz'
+  source_sha256 '1226240f8ed7f5faafbff6d93e5802c7959c4b40f9212ac6f020d67ef3aa599e'
+
+  def self.patch
+    system 'sed -i "s#-ldconfig#/usr/local/sbin/ldconfig#g" Makefile'
+  end
+
+  def self.build
+    system 'make',"PREFIX=#{CREW_PREFIX}"
+  end
+
+  def self.install
+    system 'make', "PREFIX=#{CREW_PREFIX}",
```

Comment:
Most of this is unnecessary. Just use `system 'make', "PREFIX=#{CREW_PREFIX}", "LIBDIR=#{CREW_LIB_PREFIX}", "DESTDIR=#{CREW_DEST_DIR}", 'install'`.

Reply:
In some cases, `"MANPATH=#{CREW_DEST_PREFIX}/share/man/man1/"` is needed if the default path doesn't include `share`.  He may not need `"LIBDIR=#{CREW_LIB_PREFIX}"` depending on if there are library files installed.